### ISSUE TITLE
Add actor age-at-time to Full Cast page

### ISSUE THREAD

**Issue by lortza** [mikevallano/tmdb-moviequeue#249]
I frequently find myself wondering how old an actor was when a movie was made. Adding a little math to their listing on the full cast page should be  an easy lift. It may require a light redesign. I'm game for that.

**Comment by mikevallano** [comment]
You checked and the birthdate is not included in the cast response, but we should check to see if we can send one request with multiple actor ids rather than one request per actor

**Comment by lortza** [comment]
I think I have a compromise that can move us forward for the moment. On the actor bio page there are movie credits that show a year and we know the actors birth year. We can do the math and show it in the movie credit.

**Comment by lortza** [comment]
I was not able to find any sort of group id endpoint for actor bios. :( So I'm not really sure what our next move would be here... 🤔 assuming we're not trying to persist any actor information in our own db.

**Comment by mikevallano** [comment]
Yeah, bummer. Maybe we could make a feature request. Seems like it should be relatively easy for them to add one more piece of data (`birthday`) to the https://developers.themoviedb.org/3/movies/get-movie-credits response. 

Looks like the feature request could be made in the forum:
https://www.themoviedb.org/talk/category/5047958519c29526b50017d6
